hoodie-bench/src/main/java/com/uber/hoodie/integrationsuite/job/ThreadPoolService.java

```diff
@@ -0,0 +1,65 @@
+package com.uber.hoodie.integrationsuite.job;
+
+import com.uber.hoodie.utilities.exception.HoodieWorkloadManagerException;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.stream.Collectors;
+
+/**
+ * Thread pool service for common helper methods
+ */
+public class ThreadPoolService {
```

Comment:
can we avoid this class by using either https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ExecutorService.html#invokeAll-java.util.Collection- or parallelStreams?